sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html

```diff
@@ -44,54 +44,362 @@
 var wtu = WebGLTestUtils;
 var gl = wtu.create3DContext();
 
-function rgba8_f(buf, index) {
-  return [buf[index]/255.0, buf[index + 1]/255.0, buf[index + 2]/255.0, buf[index + 3]/255.0];
+function rgba8(buf, index, expectedColor) {
+  var scale = 1.0 / 0xFF;
+  var color = [buf[index] * scale, buf[index + 1] * scale, buf[index + 2] * scale, buf[index + 3] * scale];
+  var tol = 2.0 * scale;
```

Comment:
Please think a little about how the tolerances might be more easily maintained if it turns out that some GPUs need them increased.